### DISCUSSION TITLE
When would 2024 version would be released ?

### DISCUSSION THREAD

**Discussion by Nosenzor** [cnr-isti-vclab/PyMeshLab#411] [General]
2023.12 is just one year old. Let's celebrate with a new version ;-) . I'm interested so i can run pymeshlab on Mac Arm using an official release  :-)

**Comment by alemuntoni** [comment]
pymeshlab 2023.12.post2 was released also for mac arm. You should be able to just run pip install pymeshlab (at least python 3.10 is required).
A new release was planned a month ago, but right now we don't have a certificate to sign the releases. We are waiting for a new one, but I don't know when we will be able to get it.
